README.adoc

```diff
@@ -1,15 +1,21 @@
 = Asciidoc Splitter
 
-A Java utility to split asciidoc files into a modular documentation layout.
-At some point it will also include a way to run this via maven, and possibly gradle, however, at this point in time it is CLI only.
+Java utility that split asciidoc files into a modular documentation layout.
+At some point it will also include a way to run this via maven, and possibly
+gradle, however, at this point in time it is CLI only.
 
-== Usage
+== Building and Running
 
-Build using `mvnw install -DskipTests` (until it's in a repo) then run the uberjar in `target` named `asciidoc-splitter-1.0-SNAPSHOT.jar` by using the java command:
-`java -jar target/asciidoc-splitter-1.0-SNAPSHOT.jar`.
-You will need to specify two options:
+Build using `./mvnw install -DskipTests` (until it's in a repo). This creates an
```

Comment:
@LightGuard hi Jason. this is a cool utility. I've just taken it for a quick test drive. tbh I just dove straight into the readme without looking at the contents of the repo. when I first ran I used "mvn" thinking "mvnw" was a typo. it did work but returned some WARN messages.

perhaps the above change might help others avoid a similar fate. cheers.

Reply:
Fair enough